fluent/src/resource.js

```diff
@@ -55,7 +56,7 @@ const MAX_PLACEABLES = 100;
 /**
  * Fluent Resource is a structure storing a map of parsed localization entries.
  */
-export default class FluentResource extends Map {
+export default class FluentResource extends Set {
```

Comment:
Beyond updating the comment just a line above, can we consider Array vs. Set here? We won't likely be looking for duplicate-free set, so why not Array?

Reply:
Good idea, let me test it with `make perf`.

Reply:
Sweet, extending `Array` gives a slight speed up in `addResource`. I guess iterating over the elements of an array is faster than of a set. Thanks for the suggestion!

Reply:
well, Set does more - it has to hash and look for duplicates on insertion.

Reply:
I know, but interestingly, there's no noticeable difference on the benchmark which inserts. Instead, it's the one which iterates over the elements which sped up :)